README.md

```diff
@@ -7,7 +7,7 @@ Add pdf file to parse in /ignore folder then run:
 
 ```bash
 docker build -t ciprs-reader .
-docker run --rm -v "${PWD}:/usr/src/app" ciprs-reader python ciprs-reader.py ignore/cypress-example.pdf
+docker run --rm -v /$(pwd):/usr/src/app ciprs-reader python ciprs-reader.py ignore/cypress-example.pdf
```

Comment:
This is the only way I could get the command to work across both Windows Git Bash and Linux bash